app/views/tasks/show.html.erb

```diff
@@ -21,9 +21,10 @@
   </h4>
   <% end %>
   </div>
-<div class="col-md-8">
+<div class="col-md-12">
```

Comment:
And you didn't manage to succeed at moving the button to a different block, right?